packages/frames.js/src/middleware/types.message.ts

```diff
@@ -0,0 +1,142 @@
+interface User {
```

Comment:
can we move all the neynar files into a `neynar/` directory which contains an index.ts which exports all the necessary methods and types?